src/diffpy/cmi/packsmanager.py

```diff
@@ -113,6 +113,24 @@ def available_examples(self) -> dict[str, List[tuple[str, Path]]]:
                         )
         return examples_dict
 
+    def copy_examples(
+        self,
+        user_input: Union[str, List[str]],
+        target_dir: Path = None,
+    ) -> List[Path]:
+        """Copy examples or packs into the target or current working
+        directory.
+
+        Parameters
+        ----------
+        user_input : str or list of str
```

Comment:
I suggest that we call this `examples_to_copy` or something like that.  I suggest to accept just a list of strings.